### ISSUE TITLE
What happens when display limit is reached?

### ISSUE THREAD

**Issue by TomWFox** [takecian/SwiftRater#21]
On Apple's developer documentation for SKStoreReviewController it says "the prompt will only be displayed to a user a maximum of three times within a 365-day period".

What is the procedure with this? - say if you call SwiftRater.rateApp() does it just take the user to the app's page on the App Store rather than showing the prompt?

**Comment by takecian** [comment]
Currently SwiftRater shows SKStoreReviewController only once if condition met.

**Comment by TomWFox** [comment]
So if you have a button that calls SwiftRater.rateApp() the second time it is pressed it does nothing?

**Comment by takecian** [comment]
`rateApp()` is just a wrapper for SKStoreReviewController for iOS 10.3 or later devices.
https://github.com/takecian/SwiftRater/blob/master/SwiftRater/SwiftRater.swift#L156

**Comment by TomWFox** [comment]
Ah, thanks for clearing that up.

I think it would be nice to have some kind of fallback behaviour for `rateApp()` - because if it is being used as a user selected button you always want the user to be able to leave a review so I think it should open the app's page on the App Store if `SKStoreReviewController` cannot be displayed. Do you agree?

**Comment by takecian** [comment]
Hmm, actually `SKStoreReviewController` is not for user interaction.
>it's not appropriate to call it in response to a button tap or other user action.
https://developer.apple.com/documentation/storekit/skstorereviewcontroller/2851536-requestreview

It would be good to change behavior of implementation of `rateApp()`.

**Comment by takecian** [comment]
I will change code, if rateApp() called, open review page of product url on AppStore to always interact.

**Comment by takecian** [comment]
I have updated `1.4.0`.
It navigate users to appstore review page when `rateApp()` called.
SwiftRater shows `SKStoreReviewController` if `check` called and condition met. (This does not happen frequently so i guess it does not reach limit)

**Comment by TomWFox** [comment]
Yep that makes sense, I'll check out the update, thanks!

**Comment by TomWFox** [comment]
Just updated, when I call `rateApp()` on button tap I get the warning:
```
Warning: Attempt to present <UIAlertController: 0x7fe00c11de00> on <Surprises.IntroViewController: 0x7fe00a6077e0> whose view is not in the window hierarchy!
```
And the message:
```
[SwiftRater] Trying to show review request dialog.
```

I'm assuming the source of the problem is:
https://github.com/takecian/SwiftRater/blob/cebed652b6b72f800d01ec6e62dc8384c72f3dbf/SwiftRater/SwiftRater.swift#L156

**Comment by takecian** [comment]
please try
```Swift
@IBAction func tapped() {
   rateApp(host: self)
}
```

**Comment by TomWFox** [comment]
Yep, fixed 🙃

**Comment by takecian** [comment]
Thanks. I have also updated README.
If you are okay, please close this.
